index.bs

```diff
@@ -12521,6 +12518,19 @@ values passed to \[[Call]]:
 </ol>
 
 
+<h4 id="preventextensions">Platform object \[[PreventExtensions]] method</h4>
+
+The internal \[[PreventExtensions]] method of every [=platform object=] |O|
```

Comment:
Ah yeah, it looks like things are just confused there :(. It looks like all of them have this problem, where they actually handle all cases, but their intro sentences say they only apply for indexed/named situations. Maybe let's do PreventExtensions "right" in this PR, then as a follow-up fix the others? Fixing the others would mean:

- correcting their intro sentences to just say e.g. "When the [[DefineOwnProeprty]] internal method of a platform object _O_ is called with property key _P_, the following steps are taken:"
- Using the correct OrdinaryX() fallbacks instead of the various different ways of falling back that are currently in place. (E.g. [[Set]] could replace steps 2 and 3 with a call to OrdinarySet; [[Delete]] could replace steps 3 and 4 with OrdinaryDelete; etc.)